style.css

```diff
@@ -1,8 +1,92 @@
 body {
-  box-sizing: border-box;
   margin: 0;
+}
+
+* {
+  box-sizing: border-box;
+}
+
+.nav__bar {
+  background: #64b5f6;
+  color: white;
+  overflow: auto;
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 0 2rem 0 2rem;
```

Comment:
I think you may have wanted padding here? The background should go all the way to the edge of the screen.